pkg/rpc/server/testdata/test_contract.go

```diff
@@ -54,7 +54,7 @@ func Transfer(from, to []byte, amount int) bool {
 	toBalance += amount
 	storage.Put(ctx, to, toBalance)
 
-	runtime.Notify("transfer", from, to, amount)
+	runtime.Notify("Transfer", from, to, amount)
```

Comment:
Not a 83a3681611989e6679688098568cd557eb6848ea change.

Reply:
Fixed.